Assets/Code/Scripts/Player/WalkingState.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using UnityEngine;
+
+namespace Controls {
+    /// <summary>
+    /// TODO: Make note of any music plugins we need here...
+    /// </summary>
+    public class WalkingState : PlayerState
+    {
+        // TODO: Adjust multiplier values here
+        private float _speedMultiplier = 1f;
+        private float _heartRateMultiplier = 3f;
+        
+        public void OnEnter(PlayerState prevState)
+        {
+            // TODO: When the Player walks...what should happen? music? visual animations? Does it matter from which
+            // state?
+        }
+        
+        public void Movement(Transform player, HRGauge heartRate, Action playerDeath, float baseSpeed)
+        {
+            float speed = baseSpeed + _speedMultiplier;
```

Comment:
did you mean to add or multiply here?

Reply:
Ah I meant to add here but made the variable name a little misleading so I'll adjust that